tests/Behat/AuthenticationContext.php

```diff
@@ -147,4 +147,18 @@ public function allowExternalUsers(): void {
     $this->configContext->setConfig('oe_authentication.settings', 'assurance_level', 'LOW');
   }
 
+  /**
+   * Clicks/presses a link or button.
+   *
+   * Drupal 9.3 uses Seven theme but Drupal 9.4 Claro.
+   *
+   * @todo can be removed after Drupal 9.3 support.
+   *
+   * @When I tap :label
+   */
+  public function tapElement($label): void {
+    $element = $this->getSession()->getPage()->find('xpath', "//a[text()='$label']|//input[@value='$label']");
+    $element->click();
+  }
+
```

Comment:
I don't think this is needed. There is a step to click on links that can replace the `press button` step.